tools/dbTools.py

```diff
@@ -18,11 +18,20 @@ def __init__(self, database_url=None):
         self.logger = logging.getLogger(__name__)
         if not database_url:
             database_url = os.getenv('DATABASE_URL')
-        self.connection = psycopg2.connect(database_url)
+            database_user = os.getenv('DATABASE_USER')
+            database_password = os.getenv('DATABASE_PASSWORD')
```

Comment:
You’re repeating an existing pattern, so you may not want to change it here, but it would be better to move the config parsing out of here. This is very tied to a model of using environment variables – it would be better to add constructor arguments to `DatabaseConnector`, and let the caller decide how to get the config.

Then this becomes easier to test, and less brittle to environment changes.

Reply:
You're right, I'll change it 👍